modules/integration/tests-ui-integration/tests-es-ui/src/test/java/org/wso2/carbon/greg/test/controller/Controller.java

```diff
@@ -0,0 +1,94 @@
+/*
+* Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+package org.wso2.carbon.greg.test.controller;
+
+import org.testng.TestNG;
+import org.testng.annotations.BeforeSuite;
+import org.testng.xml.XmlClass;
+import org.testng.xml.XmlSuite;
+import org.testng.xml.XmlTest;
+import org.wso2.greg.integration.common.utils.GREGIntegrationUIBaseTest;
+
+import java.util.ArrayList;
+import java.util.List;
+/***
+ * This class will create dynamic testng.xml suites and submit for testing.
+ * Loop the given test suites
+ * Original report will stored inside (tests-es-ui/target/test-output)
+ * To enable this class add this class into testng.xml and remove other classes from testng.xml
+ * put commandline arguments as ( -Diterations=10 ). 10 will be your iteration count
+ */
+public class Controller extends GREGIntegrationUIBaseTest {
+    @BeforeSuite(groups = "wso2.greg")
+    public void testController() throws InterruptedException {
+        //Create an instance on TestNG
+        TestNG testNG = new TestNG();
+        List<XmlSuite> testSuites = new ArrayList<>();
+        List<XmlTest> testsList;
+        List<XmlClass> testClasses;
+        XmlSuite testSuite;
+        String regex = "[0-9]+";
```

Comment:
Thank you Sameera for suggestion,
This has been fixed with https://github.com/wso2/product-greg/pull/847